packages/core/src/components/CopyTextButton/CopyTextButton.tsx

```diff
@@ -84,7 +73,7 @@ export const CopyTextButton = (props: Props) => {
     <>
       <textarea
         ref={inputRef}
-        style={{ position: 'absolute', top: -9999, left: 9999 }}
+        style={{ position: 'absolute', top: -9999, left: -9999 }}
```

Comment:
if it's to the right of the viewport, it adds a massive scroll bar

Reply:
Doh. Thanks for that fix. Could that have been what I was struggling with too, perhaps. I managed to get rid of the scrollbar with this little tweak: https://github.com/backstage/backstage/pull/4581/commits/23dfbf2934bb698649c4d2fcb88cd23f132cbc8c

Reply:
Heh, yeah, you can probably revert that after this is merged.

Reply:
will do that along with the improved error handling, then :)